src/exampleTypes/FacetSelect.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import F from 'futil-js'
+import _ from 'lodash/fp'
+import { contexturify } from '../utils/hoc'
+import Async from 'react-select/lib/Async'
+
+let getOptions = node =>
+  _.map(({ name }) => F.autoLabelOption(name), node.context.options)
```

Comment:
Auto label isn’t right here, but we can support a display function like in facet

Reply:
Also don’t forget to show counts and cardinality somewhere since facet gets it

Reply:
counts and display functions are in 👍

Regarding cardinality -- as far as I can tell, there's no way to customize how many results react-select will show at a time, which means no "view more" functionality on FacetSelect. Of course there's nothing stopping us from adding cardinality anyway, but it'd just kinda be sitting there by itself, and I dunno if it's useful enough on its own to justify the space it would take up. (Personally, I'm inclined to say it's not -- the best thing about FacetSelect is how compact it is)

Reply:
View more/less would still matter because it’d affect how many options are actually loaded. It’s not a react select customization - it would control the facet node via contexture - react select will just render al the options we return, so we control the size effectively that way. Cardinality (and maybe even the view more control) could be done with the group rendering thing from the first example in customizing built ins: https://react-select.com/advanced#replacing-builtins